### PR TITLE
Simple comment popover.

### PR DESCRIPTION
**Pull request by dnil**
For #588: Like so
screen shot 2017-09-01 at 19 22 59

### CODE REVIEW

**Comment by moonso** [comment]
Wow, great work @dnil !

File: scout/server/templates/utils.html [hunk 1]
```diff
@@ -11,6 +11,38 @@
   {% endwith %}
 {% endmacro %}
 
+{% macro comments_table(institute, case, comments, variant_id=None) %}
```
Comment (robinandeer) [inline scout/server/templates/utils.html]: nice use of macro!

File: scout/server/blueprints/variants/templates/variants/variants.html [hunk 2]
```diff
@@ -130,17 +130,23 @@ <h4 class="panel-title">
...
   {% elif comment_count > 0 %}
-    <!-- At least show if there are any comments written -->
-    <span class="badge pull-right">
+    <a href="#" 
+       class="badge pull-right" 
+       data-toggle="popover" 
+       data-placement="right" 
+       data-html="true" 
+       data-trigger="hover click" 
+       data-content="{{ comments_table(institute, case, variant.comments, variant._id) }}"
```
Comment (robinandeer) [inline scout/server/blueprints/variants/templates/variants/variants.html]: hehe I didn't even know we loaded comments when we displayed the variants list 😅 -- kind of pointless unless we show them.........

**Comment by moonso** [comment]
Nice I'll merge this then

**Comment by dnil** [comment]
Thanks! :blush:

**Pull request merged**